### ISSUE TITLE
Set role="presentation" on the share icon and the chevron icon

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#2032]
Non text content does not have an accessible name.

* Issue:  The WAI-ARIA image, created with role="img", does not have an accessible name.
* Snippet: `<svg class="svg-inline--fa fa-share-alt fa-w-14 mrgn-lft-sm" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="share-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" data-fa-i2svg="" style="z-index: 2147483646; position: relative;">`

How to fix

* If the image is informative or functional, use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to create an alternative text for the image.
* If the image is purely decorative, use the WAI-ARIA attribute role="presentation" instead of role="img" for the image. This will cause assistive technologies to ignore the image.

![Screen Shot 2019-03-21 at 11 57 57 AM](https://user-images.githubusercontent.com/5498428/54765842-99600b80-4bd0-11e9-93e7-6c9cf0072ddb.png)
